### PR TITLE
Bundle 0.9.0

### PR DESCRIPTION
**Pull request by SteveMattar**
fix(Makefile): bundle target
update kustomize and bundle configurations
create bundle 0.9.0

### CODE REVIEW

File: pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func main() {
 	//    type: string`
 
 	re := regexp.MustCompile("`([^`]*)`")
-	reNS := regexp.MustCompile(`(?s)\s*namespace:\s*placeholder\s*`)
+	reNS := regexp.MustCompile(`(?s)\s*\$(SA_NAMESPACE):\s*placeholder\s*`)
```
Comment (mangelajo) [inline pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go]: I think this is not the same.

The previous regex was looking for 
namespace: placeholder


Now you are looking for 

$(SA_NAMESPACE): placeholder

This needs testing, but what about:

```suggestion
	reNS := regexp.MustCompile(`(?s)\s*namespace:\s*\$(SA_NAMESPACE)\s*`)
```

**Review by mangelajo — CHANGES_REQUESTED** [review]

File: pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go (hunk 1 above)
Comment (SteveMattar) [reply pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go]: Thanks for paying attention :+1:

**Review by dfarrell07 — APPROVED** [review]

File: pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go [hunk 2]
```diff
@@ -98,7 +98,7 @@ func main() {
 	//    type: string`
 
 	re := regexp.MustCompile("`([^`]*)`")
-	reNS := regexp.MustCompile(`(?s)\s*namespace:\s*placeholder\s*`)
+	reNS := regexp.MustCompile(`(?s)\s*namespace:\s*\$(SA_NAMESPACE)\s*`)
```
Comment (skitt) [inline pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go]: I’ll check but I wonder whether this matches — going by the start of the regex, parentheses are the grouping operator, and need to be escaped to match literal parentheses.

Comment (SteveMattar) [reply pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go]: yup, I forgot to escape the parentheses as well

File: config/default/manager_auth_proxy_patch.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: submariner-operator
```
Comment (mangelajo) [inline config/default/manager_auth_proxy_patch.yaml:7]: Whats this deployment in system namespace for?

File: config/default/manager_webhook_patch.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: controller-manager-webhook
```
Comment (mangelajo) [inline config/default/manager_webhook_patch.yaml:5]: shouldn't this be submariner-operator-webhook then?

**Review by mangelajo — CHANGES_REQUESTED** [review]
A few questions about the webhooks

File: config/default/manager_auth_proxy_patch.yaml (hunk 3 above)
Comment (SteveMattar) [reply config/default/manager_auth_proxy_patch.yaml:7]: this name is just a placeholder and can be replaced when running kustomize. However, this yaml file is not included - see `config/default/kustomization.yaml`

File: config/default/manager_webhook_patch.yaml (hunk 4 above)
Comment (SteveMattar) [reply config/default/manager_webhook_patch.yaml:5]: Same here this webhook is not part of our build. we are not using webhooks at this moment, maybe in the future.

**Comment by nyechiel** [comment]
@mangelajo can you review again?

**Review by mangelajo — APPROVED** [review]
Thanks for the explanations Steve!

**Pull request merged**